src/guc.c

```diff
@@ -43,6 +43,7 @@ bool ts_guc_enable_chunk_append = true;
 bool ts_guc_enable_parallel_chunk_append = true;
 bool ts_guc_enable_runtime_exclusion = true;
 bool ts_guc_enable_constraint_exclusion = true;
+bool ts_guc_enable_contagg_pushdown = true;
```

Comment:
this lacks proper GUC definition, should probably also add a test that the GUC works